deepTumour.wdl

```diff
@@ -3,26 +3,58 @@ version 1.0
 
 workflow deepTumour {
     input {
-        File vcf_file
+        File inputVcf
+        File inputVcfIndex
+        File? inputMaf
         String outputFileNamePrefix     
         String reference
-        Boolean filterVcf
+        String filter_method
+        Array[String]? valid_exonic 
```

Comment:
I'd suggest providing a default value for both valid_exonic and exlude_mutations, that could be overwritten.  I would also be okay at making this a task-level Input, (not exposed at the top level),...overriding can still be done

Reply:
I have provided default value for these parameters. and you are right they are more like task level inputs, so removed from workflow inputs.